src/website/shared/management/commands/update_groups.py

```diff
@@ -0,0 +1,77 @@
+import argparse
+import logging
+from typing import Any
+
+from django.contrib.auth.models import Group, User
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from shared.templatetags.gh_tags import get_gh_organization, get_gh_team
+
+# TODO: move to shared.auth or shared.auth_utils
+
+logger = logging.getLogger(__name__)
+
+
+def get_team_member_ids(orgname: str, teamname: str) -> set[int]:
```

Comment:
Any reason not to use `@functools.lru_cache` with an appropriate size?

Reply:
When I was mentioning caching I was thinking about database caching and getting the info from there, but I see that you meant memoization.

I'll implement it with functools decorators. I think to make it work the memoized function will have to be triggered by a signal so it's always executed by the running webserver process, otherwise the command will not profit from the memoized cache.

Then the systemd timer can call this command by invalidating first the cache.

As an alternative to the systemd timer, I saw that cachedtools has timed versions of functools caches with `TTLCache`. I haven't read the implementation yet to see the tradeoffs, and it would also mean configuring the TTL at settings level, which seems less flexible that changing the refresh timer.